### DISCUSSION TITLE
Black History Month

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#88] [Discuss a Topic]
If you feel up to it, I would love to hear a fact or two in each class about Black History and current cultural things like the bonnet y'all shared the other day. :) I feel super lucky and honored to have 3 black women instructors/leaders at NSS.

**Comment by drteresavasquez** [comment]
We wake up like this it's true and it's not easy to maintain all this Black girl magic! 😊 

I look forward to the facts you bring! Would be good to share what you're learning about diversity for sure. It's definitely a part of the tech space.
